### PR TITLE
Fixed post request to webhook

### PR DESCRIPTION
**Pull request by karpp**

### CODE REVIEW

File: default/main.py [hunk 1]
```diff
@@ -51,8 +51,14 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
...
                     json = await resp.json()
+                    json['api_key'] = user.apikey
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    httpSession.post(user.webhook, data=json)
+                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
+                        post_response = await post.text()
+                        if post.status != 200:
+                            print(post.status)
```
Comment (niedziol) [inline default/main.py:61]: Remove this print, or change to logging.

File: default/main.py [hunk 2]
```diff
@@ -51,8 +51,14 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
...
+                    json['api_key'] = user.apikey
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    httpSession.post(user.webhook, data=json)
+                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
+                        post_response = await post.text()
+                        if post.status != 200:
+                            logging.error(post.status)
+                            logging.error(post_response)
```
Comment (niedziol) [inline default/main.py:62]: I think it should be one log msg.
Imagine when system is running and many users are making requests, then if one receives 403 and the other one 400 you can get logs this way: 403, 400, response, response and you can;t be easly sure which is which.

File: default/main.py [hunk 3]
```diff
@@ -51,8 +51,14 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
...
                     ) as resp:
                     json = await resp.json()
+                    json['api_key'] = user.apikey
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    httpSession.post(user.webhook, data=json)
+                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
+                        post_response = await post.text()
+                        if post.status != 200:
```
Comment (mgmacias95) [inline default/main.py:60]: Should this be `post_response.status` instead?

Comment (karpp) [reply default/main.py:60]: No, that's correct... Maybe it's not a good naming

**Pull request merged**